server/src/thirdparty/event.js

```diff
@@ -0,0 +1,31 @@
+import EventEmitter from 'events';
+
+const tpEventEmitter = new EventEmitter();
```

Comment:
In this file, there are multiple async functions but they don't have any await inside them. Can you please explain this ?

Reply:
My bad. Had async changes before, while refracting forgot to remove